docker/mvp-dapp/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:lts-hydrogen
+
+WORKDIR /home/app
+COPY . /home/app
+
+RUN npx lerna bootstrap
+RUN npx lerna run build
+
+WORKDIR /home/app/packages/samples/mvp-dapp
+
+CMD [ "node", "./dist/main.js" ]
```

Comment:
Is it possible to start the **mvp dapp** by `lerna task` with `scope`.

I'm not sure about it because `mvp-dapp` is in a subdirectory of `packages/samples`

Ref: https://lerna.js.org/docs/features/run-tasks